types/types.ts

```diff
@@ -167,6 +185,14 @@ export interface MainnetContracts {
   optimisticTimelock: OptimisticTimelock;
   optimisticTimelockArbitrum: OptimisticTimelock;
   curve3Metapool: IERC20;
+  arbitrumOptimisticTimelock: OptimisticTimelock;
+  arbitrumScalingPriceOracle: ScalingPriceOracle;
```

Comment:
probably a nit because they're the same really but shouldn't this be `L2ScalingPriceOracle`

Reply:
good catch